src/backend/tests/models/charge_types/test_contempt_of_court.py

```diff
@@ -0,0 +1,70 @@
+from datetime import date as date_class
+from expungeservice.models.expungement_result import EligibilityStatus
+from expungeservice.models.charge_types.contempt_of_court import ContemptOfCourt
+
+from tests.factories.charge_factory import ChargeFactory
+from tests.factories.case_factory import CaseFactory
+from tests.models.test_charge import Dispositions
+
+def test_contempt_of_court_dismissed():
+    charge_dict = {
+        "case": CaseFactory.create(type_status=["Contempt of Court", "Closed"]),
+        "name": "contempt of court",
+        "statute": "33",
```

Comment:
The first two digits are just the chapter, and it looks like the typical statutes for contempt are 33065, 33055, and 33015, so we can use those in our test cases. 

Chapter 33 https://www.oregonlaws.org/ors/chapter/33 covers a bunch of non-charge related stuff, but it seems possible there would be other charges in there that aren't contempt.